src/utils/validate.js

```diff
@@ -131,17 +132,13 @@ export type Validate = (schema: Object, values: Object, options?: ValidationOpti
  * Export `validate`.
  */
 
-export default function validate(schema: Object, values: Object, options?: ValidationOptions): FieldErrors {
-  const { keywords, ...restOptions } = options ?? {};
-  const ajv = new Ajv(merge({}, restOptions, { $data: true, allErrors: true }));
+export default function validate(schema: Object, values: Object, options?: ValidationOptions = {}): FieldErrors {
+  const ajvOptions = {
+    strict: false,
```

Comment:
Do you think this makes sense?

Reply:
No, let's keep the default. It'll make it easier to upgrade other projects.